tx-source/dash_core.php

```diff
@@ -326,6 +326,7 @@
 $Definition['Enable Pocket Locations'] = 'Enable Pocket Locations';
 $Definition['Enable Private Communities'] = 'Enable Private Communities';
 $Definition['Enable Product Integration'] = 'Enable Product Integration';
+$Definition['Enable follow up notifications.'] = 'Enable follow up notifications.';
```

Comment:
Notice the two strings above use title case without a period, while you use sentence case.

Reply:
Those strings above are related to other places of the Dashboard, not the categories add/edit:

![Screen Shot 2020-08-06 at 2 04 36 PM](https://user-images.githubusercontent.com/31521020/89566263-db9de300-d7ed-11ea-8944-62214921dcb5.png)

I was following the pattern specific to this page.